test/integration/searchRanking.js

```diff
@@ -17,45 +19,42 @@ function makeSearchRequestAndCheckExpectations(search, assertions) {
     .end(assertions);
 }
 
-function rankTop3Results(res, className) {
+function rankTopResults(res, className) {
   const $ = cheerio.load(res.text);
   const searchResults = [];
-  for (let i = 0; i < 3; i++) {
+  for (let i = 0; i < RESULTS_THRESHOLD; i++) {
     searchResults.push($(className, $('.results__item--nearby .results__details').eq(i)).text().trim());
   }
   return searchResults;
 }
 
 function expectHighRankForName(res, expected) {
-  const searchResults = rankTop3Results(res, '.results__name');
-  let highRank = false;
-  if (searchResults.indexOf(expected) > -1) {
-    highRank = true;
-  }
-  expect(highRank).to.equal(true);
+  const searchResults = rankTopResults(res, '.results__name');
+  const highRank = (searchResults.indexOf(expected) > -1);
+  expect(highRank).to.equal(true, `expected '${expected}' in top ${RESULTS_THRESHOLD} results (${searchResults})`);
 }
 
 function expectHighRankForDoctor(res, expected) {
-  const searchResults = rankTop3Results(res, '.results__gp');
+  const searchResults = rankTopResults(res, '.results__gp');
   let highRank = false;
```

Comment:
I like the highRank update in the in the method above, can you update this one to do the same please